### ISSUE TITLE
Use images.AutoOrient filter

### ISSUE THREAD

**Issue by nicokaiser** [nicokaiser/hugo-theme-gallery#51]
Instead of trying to calculate the image orientation, use the new `images.AutoOrient` filter from Hugo 0.121.2.
https://gohugo.io/functions/images/autoorient/

```diff
-       {{ $thumbnail := ($image.Fit (printf "%s %s" "600x600" .rotation)) }}
-       {{ $full := ($image.Fit (printf "%s %s" "1600x1600" .rotation)) }}
+       {{ $thumbnail := $image.Fit "600x600" | images.Filter images.AutoOrient }}
+       {{ $full := $image.Fit "1600x1600" | images.Filter images.AutoOrient }}
```

**Comment by nicokaiser** [comment]
This would require to bump the minimal Hugo version.

**Comment by nicokaiser** [comment]
Fixed in v4.0.2
